src/main/java/org/jenkinsci/plugins/gitclient/CliGitAPIImpl.java

```diff
@@ -682,6 +682,20 @@ public void clean(boolean cleanSubmodule) throws GitException, InterruptedExcept
 	launchCommand("clean", cmd);
     }
 
+    /**
+     * Remove untracked files and directories, including files listed
+     * in the ignore rules.
+     *
+     * @throws hudson.plugins.git.GitException if underlying git operation fails.
+     * @throws java.lang.InterruptedException if interrupted.
+     */
+    public void clean() throws GitException, InterruptedException {
+        reset(true);
+        String cmd = "-fdx";
+
+        launchCommand("clean", cmd);
+    }
```

Comment:
I want to first clarify I'm not a Java programmer, so what I say may not make sense or not be a common practice in Java, but can't you just set a default parameter `public void clean(boolean cleanSubmodule = false)`?